CODE_OF_CONDUCT

```diff
@@ -0,0 +1,41 @@
+
+Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. For example, when participating in technical discussions, it's important to stay on topic and keep in mind that the discussion is about ideas, concepts, or processes, not about the people behind them.
+
+Be Tolerant
+----------------
+We are a community of professionals, and we conduct ourselves professionally. We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+* Violent threats or language directed against another person
+* Discriminatory jokes and language
+* Posting sexually explicit or violent material
+* Posting (or threatening to post) other people's personally-identifying information ("doxing")
+* Personal insults
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviors
+* Repeatedly participating in any of the above behaviors. In general, if someone asks you to stop, then stop.
+* Resolve Conflicts the Community Way
+
+Disagreements, both social and technical, happen all the time, and KubeVirt is no exception. It is important that we resolve disagreements and differing views constructively and with the help of the community and community processes. When our goals differ dramatically, we encourage the creation of alternative implementations, so that the community can test new ideas and contribute to the discussion.
+
+Nobody knows everything, and nobody is expected to be perfect. Asking questions avoids many problems down the road, and so questions are encouraged. Those who are asked questions should be responsive and helpful. However, when asking a question, care must be taken to do so in an appropriate forum.
+
+If you believe that someone is violating the code of conduct or if you have any questions about the code of conduct, please email the KubeVirt community team and describe the situation or your question as clearly and detailed as possible.
+
+These guidelines are based on the [Django Code of Conduct](https://www.djangoproject.com/conduct/) and are inspired by various other community codes of conduct, in particular the [OpenStack Foundation Community Code of Conduct](https://www.openstack.org/legal/community-code-of-conduct/). License: [CC BY-SA 3.0](https://creativecommons.org/licenses/by/3.0/legalcode).
```

Comment:
In the website PR https://github.com/kubevirt/kubevirt.github.io/pull/1 we agreed to use the http://contributor-covenant.org/  so please replace this file with that content.